examples/deadline/Local-Zone/python/package/lib/config.py

```diff
@@ -37,12 +37,12 @@ def __init__(self):
         # The version of Deadline to use on the render farm. Leave as None for the latest release or specify a version
         # to pin to. Some examples of pinned version values are "10", "10.1", or "10.1.16"
         # The default value of 10.1.19 is used, to match the worker AMI ID provided below
```

Comment:
nit: comment version mismatch
```suggestion
        # The default value of 10.3.1 is used, to match the worker AMI ID provided below
```